eclipse/hono
# Quarkus based HTTP protocol adapter

This PR is dependent on #2090 and #2091

To test it out you can build Hono with

```mvn clean install -Pbuild-docker-image```

for JVM, or

```mvn clean install -Pbuild-docker-image -Pnative```

for native image.

There's a branch in `packages` project that can be used for testing (this can be merged once we merge Hono changes)

https://github.com/dejanb/packages/tree/hono-quarkus

Simply, build the package

```helm package --dependency-update hono```

and run it

```helm install -n hono -f hono/profileQuarkus.yaml eclipse-hono ./hono-1.3.15.tgz```

for JVM image, or

```helm install -n hono -f hono/profileQuarkusNative.yaml eclipse-hono ./hono-1.3.15.tgz```

for the native one.

Note that this is the basis for the Quarkus support. There's still some things to implement and fix, but hopefully we can base all the future work on this.
